multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -103,6 +107,17 @@ def parse_genome_results(self, f):
                     d[k] = r_search.group(1)
             else:
                 d[k] = r_search.group(1)
+
+    for k, r in regexes_secondary.items():
+        r_search = re.findall(r, f["f"], re.MULTILINE)
+        print(r_search)
+        if len(r_search) > 1:
+            if "\d" in r:
```

Comment:
Why do you need this?

Reply:
The original regex finds only the first match, which is always present. This regex collects all matches by using `.findall` instead of `. search`.

Reply:
But why the `if "\d" in r:` ?